CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
 # Changelog
+## 1.20.4-dev
```

Comment:
Looks like this file should not be there in this PR.

Reply:
I rebased and squashed all commits. Awaiting merge.